sdk/documentintelligence/ai-document-intelligence-rest/src/utils.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { RestError } from "@azure/core-rest-pipeline";
+
+export function streamToUint8Array(stream: NodeJS.ReadableStream): Promise<Uint8Array> {
+  return new Promise<Uint8Array>((resolve, reject) => {
+    const buffer: Buffer[] = [];
```

Comment:
confusing to have the array of Buffer chunks be also called `buffer` imo

Reply:
calling it chunks :P